.github/scripts/generate_binary_build_matrix.py

```diff
@@ -17,6 +17,7 @@
 
 # NOTE: Also update the CUDA sources in tools/nightly.py when changing this list
 CUDA_ARCHES = ["11.8", "12.6", "12.8"]
+CUDA_STABLE = "12.6"
```

Comment:
How is that related to the changes in question?

Reply:
This issue: https://github.com/pytorch/pytorch/issues/145571